lib/discourse_dev/post.rb

```diff
@@ -14,7 +14,7 @@ def initialize(topic, count)
 
       category = topic.category
       @max_likes_count = DiscourseDev.config.post[:max_likes_count]
-      unless category.groups.blank?
+      unless category&.groups.blank?
```

Comment:
`unless` and `&.` don't go well together 😛 

```suggestion
      if category&.groups.present?
```